cluster-scope/base/operators.coreos.com/subscriptions/cluster-logging/subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
-  channel: stable
+  channel: stable-5.8
```

Comment:
It looks like we're making the change to `stable-5.8` globally, rather than just on the obs cluster (which is the verbage in the pr description). Is this okay?

Reply:
@larsks the title of the PR was confusing, so I renamed it to "upgrade Loki on obs and switch Logging Operator from stable to stable-5.8 for all clusters"